README.md

```diff
@@ -119,7 +120,39 @@ To make these keys available to the jwkms you will need to;
 1. Download the Amster 6.5.1 zip file from backstage
     https://backstage.forgerock.com/downloads/browse/am/archive/productId:amster/minorVersion:6.5/version:6.5.1/releaseType:full
 1. Copy the amster zip file to `forgerock-am/_binaries` and rename the file to `amster.zip`, so that the path is `forgerock-am/_binaries/amster.zip`.
-1. Run `docker-compose up -d` and wait for services to start
+#### Docker compose
+**Compose Profiles**
+> We use the directive profiles to select the enabled services to run.[Compose profiles documentation](https://docs.docker.com/compose/profiles/)
+
+**Requirements to use compose profiles**
+- Compose version >= 1.28
+- MacOs users: upgrade docker desktop to 3.2.1 or later or used `brew` to install/upgrade it.
+- Linux users [Install docs](https://docs.docker.com/compose/install/):
+  - Alternatives
+  ```shell
+    pip install docker-compose
+  ```
+  ```shell 
+    curl -L "https://github.com/docker/compose/releases/download/1.22.0/docker-compose-$(uname -s)-$(uname -m)" > ./docker-compose
+    sudo mv ./docker-compose /usr/bin/docker-compose
+    sudo chmod +x /usr/bin/docker-compose
+  ```
+**Run docker compose profiles**
+- These services without profile will be run always, we treated them like as `mandatory services`
+- Current Profiles:
+  - _**all**_: Runs `all` profile platform services
+    ```shell
+    docker-compose --profile all up
```

Comment:
Is this the default?

Reply:
I have make a rollback on docker-compose file to use current `docker-compose up` command.
I have created a new file (`docker-compose-profiles.yml` like as alternative.
I have updated the readme file.